synapse/crypto/keyring.py

```diff
@@ -592,7 +586,7 @@ def get_key(key_server):
 
             return {}
 
-        results = yield make_deferred_yieldable(
+        results = await make_deferred_yieldable(
```

Comment:
We probably can, yeah. I'm not sure there's much benefit to changing it as part of this PR?

Reply:
It looks like there's a couple of places in this file that this could be done. I think we should do this separately since it isn't really related to async/await conversion!

Reply:
Sounds good to me, yeah.